chroma-manager/chroma_core/plugins/linux.py

```diff
@@ -354,60 +354,58 @@ def preferred_serial(bdev):
         # So we have to build a graph and then traverse it to populate our resources.
         for bdev in devices['devs'].values():
             if bdev['major_minor'] in lv_block_devices:
-                node, created = self.update_or_create(LinuxDeviceNode,
-                                    host_id = host_id,
-                                    path = bdev['path'])
+                node, created = self.update_or_create(
```

Comment:
These three conditions seems to result in the same call. Could it be a single if with an || for each condition?
```
if cond1 or cond2 or cond3:
   node, created = self.update_or_create(
      LinuxDeviceNode, host_id=host_id, path=bdev['path'])
else:
   continue
```